docs/getting-started.md

```diff
@@ -0,0 +1,68 @@
+# Getting Started
+
+This getting started guide will help you bootstrap your first cluster using Crossplane Blueprints.
+
+## Prerequisites
+
+Ensure that you have installed the following tools locally:
+
+- [awscli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+- [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+- [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+## Deploy
+
+### eksctl
+
+1. TBD
+
+### terraform
+
+1. For consuming Crossplane Blueprints, please see the [Consumption]() section. 
```

Comment:
is there a purpose to having this refer to an empty link?

Reply:
fixed